decidim-core/lib/decidim/core.rb

```diff
@@ -477,7 +477,7 @@ def self.reset_all_column_information
   end
 
   # Blacklisted passwords. Array may contain strings and regex entries.
```

Comment:
We should also fix the comment above:
```suggestion
  # Denied passwords. Array may contain strings and regex entries.
```